storage/innobase/xtrabackup/src/xbcloud/xbcloud-t.cc

```diff
@@ -489,3 +493,97 @@ TEST(keystone_v3, project_scoped_with_project_domain_keystone_v3_success) {
       auth_info.url,
       "https://example.com:8080/v1/AUTH_67a0195262ec4452bd2af48e75bcb687");
 }
+
+TEST(azure_client, basicDNSv4) {
+  Mock_http_client http_client;
+  Azure_client c(&http_client, "my-storage-account", "my-access-key-id", 0,
+                 "my-storage-class", 1, 1);
+
+  EXPECT_CALL(
+      http_client,
+      make_request(
+          AllOf(
+              IsPut(),
+              Property(&Http_request::url,
+                       StrEq("https://my-storage-account.blob.core.windows.net/"
+                             "test-b?restype=container"))),
+          Response(200, "", std::vector<char *>{})))
+      .WillOnce(Return(true));
+  c.create_container("test-b");
+
+  Http_buffer buf;
+  buf.append("some contents");
+
+  EXPECT_CALL(
+      http_client,
+      make_request(
+          AllOf(IsPut(),
+                Property(
+                    &Http_request::url,
+                    StrEq("https://my-storage-account.blob.core.windows.netb/"
```

Comment:
is the .netb a typo here? or expected?

Reply:
b was for container , I have changed it to bucket.